test/integration/model.js

```diff
@@ -975,7 +964,7 @@ module.exports = function(bookshelf) {
         expect(newModel.get('created_at')).to.be.an.instanceOf(Date);
         expect(newModel.get('updated_at')).to.be.an.instanceOf(Date);
 
-        expect(existingModel.get('created_at')).to.not.exist;
```

Comment:
Here it's the same thing as above. We don't want `created_at` to change with an update.

Reply:
See my previous comment

Reply:
By the way, I'm not sure why the tests fails now. It looks like some discrepancy in 0.003ms
Need to take a deeper look at it

Reply:
Yeah, I noticed that discrepancy. I'll take a look at the failing test after the next release.